triton-inference-client.spec

```diff
@@ -90,7 +90,7 @@ cmake ${PROJ_DIR} \
     -DTRITON_VERSION=%{realversion} \
     -DCMAKE_CXX_FLAGS="-Wno-error -fPIC" \
     -DFETCHCONTENT_SOURCE_DIR_REPO-COMMON=${COMMON_DIR} \
-    -DCMAKE_PREFIX_PATH="${GRPC_ROOT};${ABSEIL_CPP_ROOT}"
+    -DCMAKE_PREFIX_PATH="${GRPC_ROOT};${ABSEIL_CPP_ROOT};${RE2_ROOT}"
```

Comment:
@iarspider , please also add an explicit dependency on `re2` here

Reply:
One more reason to move to Spack: automatic propagation of dependencies :)

Reply:
Do I need to add re2 to toolfile as well?

Reply:
yes